src/hawkmoth/Makefile.local

```diff
@@ -1,5 +1,5 @@
 # -*- makefile -*-
 
-dir := src/hawkmoth
+src_dir := src/hawkmoth
```

Comment:
Being overly pedantic here, but this would fit better with the 1st commit. No biggie though.

Reply:
I guess I made this drive-by change, and that inspired me to do the `docdir` change.